Awesome/index2.html

```diff
@@ -0,0 +1,317 @@
+                  aspernatur ex, architecto eaque quo suscipit.
+                  </p>
+                </div>
+                <div class="quotation__mark">
+                </div>
+              </div>
+              <div class="person">
+                <div class="person__photo">
+                  <img src="images\Person_1.png">
+                </div>
+                <div class="person__text-wrapper">
+                  <p>Davis Rhiel Madsen</p>
+                  <p>Manager</p>
+                </div>
+              </div>
+          </div>
+          <div class="comment__block2">
+            <div class="quotation">
+              <div class="quotation__text">
+                <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Doloribus accusamus expedita repellat similique odio
+                aspernatur ex, architecto eaque quo suscipit.
+                </p>
+              </div>
+              <div class="quotation__mark">
+              </div>
+            </div>
+            <div class="person">
+              <div class="person__photo">
+                <img src="images\Person_2.png">
+              </div>
+              <div class="person__text-wrapper">
+                <p>Davis Rhiel Madsen</p>
+                <p>Manager</p>
+              </div>
+            </div>
+          </div>
+          <div class="comment__block3">
+            <div class="quotation">
+              <div class="quotation__text">
+                <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Doloribus accusamus expedita repellat similique odio
+                aspernatur ex, architecto eaque quo suscipit.
+                </p>
+              </div>
+              <div class="quotation__mark">
+              </div>
+            </div>
+            <div class="person">
+              <div class="person__photo">
+                <img src="images\Person_3.png">
+              </div>
+              <div class="person__text-wrapper">
+                <p>Davis Rhiel Madsen</p>
+                <p>Manager</p>
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+      <div class="design-section">
+          <h1>STYLISH AXURE DESIGN</h1>
+          <div class="design-section__line"></div>
+          <p>Use the sections you need, remove the ones you don't.  Create gorgeous prototypes faster than ever!</p>
+          <div><button>Download</button></div>
+      </div>
+      <div class="contact">
+        <div>
+          <div class="contact__tittle-wrapper">
+            <h2 class="awesome__title">contact us</h2>
+          </div>
+          <div class="awesome__line"></div>
+          <h5 class="awesome__subtitle">Lorem ipsum dolor sit amet, consectetur adipiscing elit.</h5>
+        </div>
+        <div class="contact-form">
+          <div class="contact-form__wrapper">
+            <div class="contact-form__input">
+              <p>Name</p>
```

Comment:
For input's title there is an element label.

Reply:
Corrected. Used inputs and text area.